src/plugins/intel_npu/src/plugin/npuw/partitioning/online/compiler.cpp

```diff
@@ -140,10 +140,8 @@ std::vector<Isolate> getIsolates(const std::string isolates_unparsed) {
     if (!isolates.empty()) {
         LOG_INFO("Online partitioning will isolate subgraphs containing specified patterns.");
     } else {
-        LOG_WARN("Incorect pattern in NPUW_ONLINE_ISOLATE!"
-                 << " Please, follow the example: "
-                 << "Op:Select/NPU,P:DQMatMulGQ/compute,P:DQMatMulCW/compute,P:RMSNorm/compute. "
-                 << "No isolate rules will be taken into account during partitioning!");
+        LOG_WARN("Incorect pattern in NPUW_ONLINE_ISOLATE! No isolate rules will be taken into account during "
+                 "partitioning!");
```

Comment:
Please just write what's the pattern which is wrong. Otherwise this warning is useless.
It was the point of the past comment, too, but maybe lost in translation.

Reply:
Well, all of them are wrong. This log occurs if we couldn't parse any isolates from the user-provided string